### PR TITLE
Add ScratchTools button to new editor

### PR DESCRIPTION
**Pull request by rgantzos**
https://github.com/STForScratch/ScratchTools/assets/86856959/c67f2d36-121e-4541-9b56-c7e45367f8c0

### CODE REVIEW

File: api/spaces.js [hunk 1]
```diff
@@ -197,6 +197,11 @@ ScratchTools.appendToSharedSpace = function ({ space, element, order, scope }) {
       },
       until: () => [],
     },
+    afterProfileLocation: {
```
Comment (rgantzos) [inline api/spaces.js:200]: Didn’t mean to include this, I’ll put this in a separate pr tomorrow.

**Review by Daniel4-Scratch — APPROVED** [review]
